filer/models/abstract.py

```diff
@@ -46,14 +46,11 @@ class BaseImage(File):
     )
 
     @classmethod
-    def matches_file_type(cls, iname, ifile, request):
-        # This was originally in admin/clipboardadmin.py  it was inside of a try
-        # except, I have moved it here outside of a try except because I can't
-        # figure out just what kind of exception this could generate... all it was
-        # doing for me was obscuring errors...
-        # --Dave Butler <croepha@gmail.com>
-        iext = os.path.splitext(iname)[1].lower()
-        return iext in ['.jpg', '.jpeg', '.png', '.gif']
+    def matches_file_type(cls, iname, ifile, mime_type):
+        # source: https://www.freeformatter.com/mime-types-list.html
+        image_subtypes = ['gif', 'jpeg', 'png', 'x-png', 'svg+xml']
```

Comment:
I had to change "svg" to "svg+xml" because 'svg' uses the Filer.File model and 'svg+xml' uses the Filer.Image model.
Thanks for the source www.freeformatter.com/mime-types-list.html!